src/core/redux/reducers/resetRequestStates.js

```diff
@@ -4,16 +4,19 @@
 
 import { Map } from 'immutable';
 import { RequestStates } from 'redux-reqseq';
+import { LangUtils } from 'lattice-utils';
 
 import { ERROR, REQUEST_STATE } from '../constants';
 import type { ResetRequestStatesAction } from '../actions';
 
+const { isNonEmptyString } = LangUtils;
+
 export default function reducer(state :Map, action :ResetRequestStatesAction) {
```

Comment:
Why the rename?

<img width="698" alt="Screen Shot 2021-04-29 at 10 38 23 PM" src="https://user-images.githubusercontent.com/440299/116653234-9bfac680-a93b-11eb-8b9e-aa2b35d7d3c8.png">

Reply:
I thought every reducer had the suffix, at least every one I've written. I don't have a strong opinion on this one. The only reason I use it is so that it's easier to identify sagas vs reducers when searching in the IDE / GitHub as well as in code reviews.

Reply:
yeah, I'm wrong... lol the suffix is definitely there in most places.

Reply:
added a Reducer suffix to all the reducers I added.